README.md

```diff
@@ -62,6 +62,29 @@ $ deis open
 $ deis scale web=3 -a mytest # to scale up the app
 ```
 
+### Dockerfile for deis/charts e2e tests
+```
+$ docker build .
+
+To call create script
+$ docker run -v YOUR VOLUME WITH SECRET FILE:/root/.secrets/ IMAGE_ID /bin/bash _scripts/tests/create.sh 
```

Comment:
Perhaps an explanation on what exactly should be in the secret file would be useful. I don't see any reference to it elsewhere in the README so users first trying this out won't know where to start.